src/main/java/uk/gov/hmcts/darts/datamanagement/service/impl/InboundToUnstructuredProcessorImpl.java

```diff
@@ -55,17 +56,17 @@ public void processInboundToUnstructured() {
 
     private void processAllStoredInboundExternalObjects() {
 
-        List<ExternalObjectDirectoryEntity> inboundList = externalObjectDirectoryRepository.findByStatusAndType(getStatus(STORED), getType(INBOUND));
+        List<EntityIdOnly> inboundList = externalObjectDirectoryRepository.findByStatusAndExternalLocationType(getStatus(STORED), getType(INBOUND));
```

Comment:
Hi @cmagowan-hmcts, I agree with you this would be a better approach than the existing one. But if I make this change, then I'll have to make plenty of other changes to this automated task + its tests. Which is outside the scope of this ticket. Your suggestion will consist in re-writing part of the automated task. Whereas the original scope of my ticket was to move the `@Transactional` annotation only.
I'm happy to make these changes, but I'd suggest these are done as part of a new ticket.